qemu/tests/blockdev_commit_hotunplug.py

```diff
@@ -0,0 +1,60 @@
+from virttest import utils_misc
+
+from provider import job_utils
+from provider import backup_utils
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitHotunplug(BlockDevCommitTest):
+
+    def is_device_deleted(self, device):
+        return device not in str(self.main_vm.monitor.info_block())
+
+    def commit_snapshots(self):
+        device = self.params.get("device_tag")
+        device_params = self.params.object_params(device)
+        snapshot_tags = device_params["snapshot_tags"].split()
+        self.device_node = self.get_node_name(device)
+        options = ["speed"]
+        arguments = self.params.copy_from_keys(options)
+        arguments["speed"] = self.params["commit_speed"]
+        device = self.get_node_name(snapshot_tags[-1])
+        commit_cmd = backup_utils.block_commit_qmp_cmd
+        cmd, args = commit_cmd(device, **arguments)
+        self.main_vm.monitor.cmd(cmd, args)
+        self.main_vm.monitor.cmd('device_del',
+                                 {'id': self.params["device_tag"]})
+        unplug_s = utils_misc.wait_for(lambda: self.is_device_deleted(device),
+                                       timeout=60, step=1.0)
+        if not unplug_s:
+            self.test.fail("Hotunplug device failed")
+        job_status = str(job_utils.query_block_jobs(self.main_vm))
+        if self.params["expect_status"] not in job_status:
+            self.test.fail("Job status not correct,job status is %s"
+                           % job_status)
+        job_id = args.get("job-id", device)
+        self.main_vm.monitor.cmd("block-job-set-speed",
+                                 {'device': job_id, 'speed': 0})
+        job_utils.wait_until_block_job_completed(self.main_vm, job_id)
+
+    def run_test(self):
```

Comment:
Leidong, we can also remove run_test from your script, let's follow the test steps in BlockDevCommitTest

Reply:
Since we have hotunpluged the data disk, we can't verify files md5sum here.